core/src/main/java/hudson/model/Queue.java

```diff
@@ -377,7 +377,7 @@ public void load() {
             // first try the old format
             File queueFile = getQueueFile();
             if (queueFile.exists()) {
-                try (BufferedReader in = new BufferedReader(new InputStreamReader(Files.newInputStream(queueFile.toPath())))) {
+                try (BufferedReader in = Files.newBufferedReader(queueFile.toPath(), Charset.defaultCharset())) {
```

Comment:
I wonder why we're not using UTF-8 here.

Reply:
dunno. I am fine with fixing it in a separate PR. So far I think that all admins running Jenkins with non-UTF-8 just like to live dangerously, so I just wish them good luck

Reply:
I will submit a PR to make this change. Because it will conflict with PR #3219, I will wait until PR #3219 is merged before submitting a PR to change the charset.

Reply:
I created PR #3224 to switch to UTF-8.